ads/google/a4a/traffic-experiments.js

```diff
@@ -86,17 +97,34 @@ export function googleAdsIsA4AEnabled(win, element, experimentId, branches) {
  * @param {!Window} win
  * @param {string} experimentId
  * @param {!Branches} branches
+ * @param {!string} manualId
  * @visibleForTesting
  */
-function handleUrlParameters(win, experimentId, branches) {
-  const a4aParam = /(?:\?|&)a4a=([0-9]+)/.exec(win.location.search);
+function handleUrlParameters(win, experimentId, branches, manualId) {
+  // The URL for a Google-search-served experiment looks like:
+  //   https://cdn.ampproject.org/path/to/file.html?param0=a&exp=a4a:X&param1=b
+  // where the 'exp' parameter has the form:
```

Comment:
The description can start from this line, the previous 2 lines are pretty much common sense.
You can move the description to line 111, where `exp` debuts.